src/documenti/esterni/PdQ/contenuti/miglioramento.tex

```diff
@@ -4,35 +4,35 @@ \chapter{Valutazioni per il miglioramento}
     \rowcolor[HTML]{C0C0C0}
     Organizzativo & Bassa  & Durante il periodo della seconda milestone, dal 20/12/21 al 10/01/22, il gruppo si è reso conto di non essere in grado di rispettare le ore preventivate. &Il gruppo ha consuntivato meno ore rispetto a quelle preventivate. Inoltre per evitare che tale errore possa ripetersi si è deciso di porre maggior attenzione nell'attività di previsione oraria, andando a segnalare le ore che si è sicuri verranno impiegate per l'avanzamento delle attività di progetto.  \\ \hline
     \rowcolor[HTML]{EFEFEF}
     Organizzativo & Media & Durante il periodo della terza milestone, dal 15/01/2022 al 04/02/2022, il gruppo si è reso conto di non essere in grado di rispettare le ore preventivate. La presenza della sessione d'esame ha occupato più tempo del previsto e la stima di disponibilità oraria è risultata quindi errata. & Il gruppo ha consuntivato meno ore rispetto a quelle preventivate. Per evitare che questa situazione possa ripresentarsi, visto che è già la seconda volta, il gruppo ha capito che deve prestare ancora più attenzione nell'attività di previsione oraria, cercando di prevedere quali potrebbero essere le problematiche che possono presentarsi durante il periodo.  \\
+    \rowcolor[HTML]{C0C0C0}
+    Resoconto attività di verifica & Media & Fino a poco prima della revisione RTB il gruppo non ha controllato il progresso dei documenti utilizzando le metriche che si era posto di sfuttare. In questo modo, in prossimità della revisione, è stato impossibile poter modificare in modo adeguato i documenti per renderli il più conformi possible a quanto inizialmente prestabilito. & Il gruppo ha deciso di svolgere i dovuti controlli ai documenti ad ogni termine delle milestone, indicativamente ogni due settimane, in modo da avere una visione più chiara del loro sviluppo.\\ \hline
     \hline
-    \end{tabular}
-\end{table}
+\end{longtable}
 
 
 
 \section{Valutazione sui ruoli}
-\begin{table}[H]
-    \centering
-    \begin{tabular}{|p{0.16\linewidth}|p{0.11\linewidth}|p{0.3\linewidth}|p{0.3\linewidth}|}
-    \hline
-    \rowcolor[HTML]{036400}
-    {\color[HTML]{FFFFFF} \textbf{Problema}} & {\color[HTML]{FFFFFF} \textbf{Gravità}} & {\color[HTML]{FFFFFF} \textbf{Descrizione}} & {\color[HTML]{FFFFFF} \textbf{Soluzione}} \\ \hline
-    \rowcolor[HTML]{EFEFEF}
-    &  & & \\ \hline
-    \end{tabular}
-\end{table}
+Al momento non sono stati riscontrati problemi riguardanti i ruaoli da parte del gruppo.
+%\begin{table}[H]
+%    \centering
+%    \begin{tabular}{|p{0.16\linewidth}|p{0.11\linewidth}|p{0.3\linewidth}|p{0.3\linewidth}|}
+%    \hline
+%    \rowcolor[HTML]{036400}
+%    {\color[HTML]{FFFFFF} \textbf{Problema}} & {\color[HTML]{FFFFFF} \textbf{Gravità}} & {\color[HTML]{FFFFFF} \textbf{Descrizione}} & {\color[HTML]{FFFFFF} \textbf{Soluzione}} \\ \hline
+%    \rowcolor[HTML]{EFEFEF}
+%    &  & & \\ \hline
+%    \end{tabular}
+%\end{table}
 
 
 \section{Valutazione sugli strumenti di lavoro}
```

Comment:
Aggiungerei latex per le difficoltà nel posizionamento di immagini e soprattutto tabelle nel documento che ancora adesso creano lavoro in più